### PR TITLE
R94 keane

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,10 @@
+Release 94 Ellora (2017-09-27)
+------------------------------
+Redshift: update atomic.events to use ZSTD compression (#3435)
+Redshift: added migration script for 0.8.0 to 0.9.0 (#3440)
```
Comment (alexanderdean) [inline CHANGELOG]: Should be present tense

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+Redshift: Update domain_sessionidx column in atomic.events to accept larger values (#1788)
```
Comment (alexanderdean) [inline CHANGELOG]: should be update not Update

Comment (BenFradet) [reply CHANGELOG]: this commit isn't going to be in #3438, I'll do it myself.

**Comment by BenFradet** [comment]
cherry-picked into #3438, closing

**Pull request closed without merging**